src/routes/lists.tsx

```diff
@@ -12,7 +12,14 @@ export function Lists() {
   return (
     <div className="flex flex-1 justify-center">
       <div className="flex w-full flex-col gap-4">
-        {bookmarks.length > 0 ? <h1 className="text-2xl font-bold">Your Bookmarks</h1> : null}
+        {bookmarks.length > 0 ? (
+          <h1 className="text-2xl font-bold">Your Bookmarks</h1>
+        ) : (
+          <div>
+            <h1 className="text-2xl font-bold">No Bookmarks</h1>
+            <p className="text-sm">Add some books here by clicking the bookmark icon on a book card.</p>
```

Comment:
Maybe more along the lines of: Start adding some books using the bookmark button. Start searching here <add link to search page> ?